modules/geopackage/src/lib/parse-geopackage.js

```diff
@@ -381,3 +414,54 @@ function interleaveResults(columns, values) {
 
   return merged;
 }
+
+/**
+ * Get arrow schema
+ *
+ * @param  {Database} db GeoPackage object
+ * @param  {string} tableName  table name
+ * @return {Schema | null} Arrow-like Schema
+ */
+function getArrowSchema(db, tableName) {
+  const metadata = db.exec(
+    `SELECT * FROM \`sqlite_master\` WHERE type='table' AND tbl_name='${tableName}';`
+  )[0];
+  const sql =
+    (metadata.values &&
+      metadata.values[0] &&
+      metadata.values[0][4] &&
+      metadata.values[0][4].toString()) ||
+    '';
+  let columnMatch = COLUMNS_REGEXP.exec(sql);
+  const fields = [];
+  while (columnMatch) {
+    fields.push(new Field(columnMatch[1], new SQL_TYPES[columnMatch[2]](), true));
+    columnMatch = COLUMNS_REGEXP.exec(sql);
+  }
+  return new Schema(fields);
+}
+
+/**
+ * Form RegExp for taking column names and column types from SQL CREATE TABLE request
+ *
+ * @return {RegExp} an array of objects, where each object represents a single row
```

Comment:
The description of this parameter looks wrong. This describes the action of the returned regexp rather than the return value.

Reply:
Done